spec.html

```diff
@@ -5171,7 +5177,8 @@ <h1>Abstract Equality Comparison</h1>
         1. If Type(_x_) is the same as Type(_y_), then
           1. Return the result of performing Strict Equality Comparison _x_ === _y_.
         1. If _x_ is *null* and _y_ is *undefined*, return *true*.
-        1. [id="step-abstract-equality-comparison-web-compat-insertion-point"] If _x_ is *undefined* and _y_ is *null*, return *true*.
+        1. If _x_ is *undefined* and _y_ is *null*, return *true*.
+        1. [id="step-abstract-equality-comparison-web-compat-insertion-point"] NOTE: Annex <emu-xref href="#sec-IsHTMLDDA-internal-slot-aec"></emu-xref> adds additional steps at this point.
```

Comment:
I'd prefer

```suggestion
        1. [id="step-abstract-equality-comparison-web-compat-insertion-point"] NOTE: This step is replaced in section <emu-xref href="#sec-IsHTMLDDA-internal-slot-aec"></emu-xref>.
```

to match the others. You'll have to reword the annex B section to match as well. @tc39/ecma262-editors what do you think?

Reply:
I'm fine with this suggestion.

Reply:
@michaelficarra I believe the annex b reference already matches with it